pkg/readers/github_org_reader.go

```diff
@@ -21,6 +21,7 @@ type GithubOrgReaderConfig struct {
 	IncludeArchived        bool
 	MaxRepositories        int
 	SkipDependencyGraphAPI bool
+	ExcludeRepos           string
```

Comment:
This should be `[]string` because thats what is expected by the business logic. Let the caller handle input formats